src/tests/multihost/alltests/test_services.py

```diff
@@ -88,15 +88,19 @@ def test_0003_bz1713368(self, multihost):
         """
         # sssd-dbus is a weak dependency of sssd-tools package so we did not
         # get sssd-dbus in '# yum deplist sssd-tools' command
-        rpm_grep = 'yum repoquery --recommends sssd-tools'
-        cmd = multihost.client[0].run_command(rpm_grep, raiseonerr=False)
+        version = float(re.findall("\d+\.\d+", multihost.client[0].distro)[0])
```

Comment:
prepend 'r' to regex pattern to declare it as raw string
```r'\d+\.\d+'```
 or it gives error
```<unknown>:91: DeprecationWarning: invalid escape sequence \d```

Reply:
Fixed